src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketGitSCMBuilder.java

```diff
@@ -158,7 +158,7 @@ public BitbucketSCMSource scmSource() {
     @NonNull
     public BitbucketGitSCMBuilder withCredentials(String credentialsId, BitbucketRepositoryProtocol protocol) {
         if (StringUtils.isNotBlank(credentialsId)) {
-            StandardCredentials credentials = BitbucketCredentials.lookupCredentials(
+            StandardCredentials credentials = BitbucketCredentials.lookupCredentialsAndTrackUsage(
```

Comment:
this is already tracked by the git plugin on checkout. If your pipeline does not have a `checkout scm` you never use this credentials.